src/lib/features/read-transcript/ui/specs/toggle-transcript.spec.ts

```diff
@@ -0,0 +1,5 @@
+export {};
+
+it.todo('displays the current transcript state to the user');
```

Comment:
issue (non-blocking): this will make you write two `render()` calls in the same test, which will make it grow excessively in terms of code size.

suggestion: split it into two tests — one for when the transcript is hidden and another for when it's shown

praise: wow, `it.todo()` is a thing? Didn't know that, great job discovering it